src/bosh_aws_cpi/lib/cloud/aws/instance.rb

```diff
@@ -83,6 +83,30 @@ def exists?
       @aws_instance.exists? && @aws_instance.status != :terminated
     end
 
+    def update_routing_tables(route_definitions)
+      if route_definitions.length > 0
+        @logger.debug("Associating instance with destinations in the routing tables")
+        tables = @aws_instance.vpc.route_tables
+        route_definitions.each do |definition|
+          @logger.debug("Finding routing table '#{definition["table_id"]}'")
+          table = tables[definition["table_id"]]
+          @logger.debug("Sending traffic for '#{definition["destination"]}' to '#{@aws_instance.id}' in '#{definition["table_id"]}'")
+
+          existing = false
+          table.routes.each do |route|
```

Comment:
Docs say `routes` is a normal array, so could probably shorten this to: `existing = table.routes.any ...`